canonical-web-and-design/ubuntu.com
# Replace latest news script with npm version

## Done

Replace latest news script with npm version to keep in sync between projects

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Check the following pages to see if blog posts load and that they match the equivalent page in production
  - homepage (Latest news from our blog)
  - /robotics/community (Recent news)
  - /containers (Further reading section in footer)


## Issue / Card

Fixes https://github.com/canonical-web-and-design/latest-news/issues/3
